Tests/Features/UserPreferences/Data/NotificationPreferences.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Not sure yet wether that's 1-to-1 the structure that the JSON decoder expects via `DataToDecodableMapper<UserPreferencesEntity>()`.

Reply:
When lifting the Json object out of the `notification_preferences` key I get the following error though, so I assume the root structure being correct:
```
Key 'ContainerKeys(stringValue: "notification_preferences", intValue: nil)' not found: No value associated with key ContainerKeys(stringValue: "notification_preferences", intValue: nil) ("notification_preferences").
```